tests/components/zha/test_discover.py

```diff
@@ -45,11 +61,25 @@
             ent for ent in entity_ids if ent.split(".")[0] in zha_const.COMPONENTS
         }
 
-        event_channels = {
-            arg[0].cluster_id
-            for arg, kwarg in cr_ch.call_args_list
-            if kwarg.get("channel_class") == EventRelayChannel
+        zha_dev = zha_gateway.get_device(zigpy_device.ieee)
+        event_channels = {  # pylint: disable=protected-access
+            ch.id for ch in zha_dev._relay_channels.values()
         }
 
         assert zha_entities == set(device["entities"])
         assert event_channels == set(device["event_channels"])
+
+        entity_map = device["entity_map"]
+        for calls in _dispatch.call_args_list:
+            di = calls[0][2]
```

Comment:
can we have a better variable name here? What is di?

Reply:
assuming discovery_info. Why not use the words :) makes it easier to read.

Reply:
fixed in https://github.com/home-assistant/home-assistant/pull/31191/commits/17c59cde2388d3d44ad3a62e5b7cbce57a237771